support/vm/testing.go

```diff
@@ -62,7 +63,12 @@ func NewVMWithSingletons(ctx context.Context, t testing.TB, bs ipldcbor.IpldBloc
 	store := adt.WrapBlockStore(ctx, bs)
 	vm := NewVM(ctx, lookup, store)
 
-	initializeActor(ctx, t, vm, &system.State{}, builtin.SystemActorCodeID, builtin.SystemActorAddr, big.Zero())
+	manifestDataCid, err := store.Put(ctx, &manifest.ManifestData{})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	initializeActor(ctx, t, vm, &system.State{BuiltinActors: manifestDataCid}, builtin.SystemActorCodeID, builtin.SystemActorAddr, big.Zero())
```

Comment:
instead follow the same pattern: call construct state which you have written to do this empty manifest writing for you and then call initializeActor on the systemState variable output from construcstate

Reply:
ok.

Reply:
done.